doc/user/content/get-started.md

```diff
@@ -12,54 +12,35 @@ aliases:
   - /install/
 ---
 
-[//]: # "TODO(morsapaes) Once we're GA, add details about signing up and logging
-into a Materialize account"
+This guide walks you through getting started with Materialize.
 
-This guide walks you through getting started with Materialize, covering:
+- First, you'll sign up for a free Materialize trial account and prepare your environment.
+- Next, you'll connect a sample streaming data source and explore the unique value of Materialize: computing real-time results over fast-changing data.
+- Finally, you'll simulate a failure to see active replication in action.
 
-* Connecting to a streaming data source
+Along the way, you'll learn about basic concepts like clusters and replicas and how SQL features like views, indexes, and materialized views work in the context of Materialize. 
 
-* Getting familiar with views, indexes and materialized views
+## Before you begin 
 
-* Exploring common patterns like joins and time-windowing
+Materialize is wire-compatible with PostgreSQL, which means it integrates with most client libraries, ORM frameworks, and other [third-party tools](/integrations/) that support PostgreSQL.
 
-* Simulating a failure to see active replication in action
-
-{{< note >}}
-Ready to get hands-on with Materialize? [Sign up](https://materialize.com/register/) to get started! 🚀
-{{</ note >}}
-
-## Set up
-
-### Connect to Materialize
-
-Open a terminal window and connect to Materialize using [a compatible CLI](/integrations/sql-clients/), like `psql`. If you already have a compatible CLI installed, use the connection string provided in the Materialize UI to connect:
-
-```bash
-psql "postgres://user%40domain.com@host:6875/materialize"
-```
-
-Otherwise, install `psql` using the following instructions:
+In this guide, you'll use [`psql`](https://www.postgresql.org/docs/current/app-psql.html) to interact with Materialize, so make sure you have it installed locally.
 
 {{< tabs >}}
-{{< tab "macOS">}}
+{{< tab "Mac">}}
```

Comment:
Nit: `Mac` is the brand of computer hardware; `macOS` is the OS it runs. Since we're saying `Linux` and `Windows` below (as opposed to Thinkpad, Dell, etc.) I think we should also say `macOS` here for consistency.